docker-compose.yml

```diff
@@ -71,6 +71,7 @@ services:
       - ./permissions_engine/permissions.rego:/policy.rego
       - ./permissions_engine/idp.rego:/idp.rego
       - ./permissions_engine/authz.rego:/authz.rego
+      - ./permissions_engine/ga4ghPassport.rego:/ga4ghPassport.rego
```

Comment:
So we won't be using this ga4ghPassport.rego file, we're not using REMS for authorization